wazuh/wazuh-kibana-app
# Rename vulnerability.under_evaluation field

### Description
This pull request renames the `vulnerability.under_evaluation` field.
 
### Issues Resolved
#7045 

### Evidence

![image](https://github.com/user-attachments/assets/16a43e96-2480-4d3a-ad20-ef0b9fea257c)

![image](https://github.com/user-attachments/assets/3bb7335c-d9b6-4580-95d0-ee05e439a8c3)


### Test

- Load vulnerabilities sample data
- Navigate to Vulnerabilities module / Dashboard
- Verify the Pending evaluation KPI has data
- Use the Under evaluation custom filter and check it filters
- Navigate to Vulnerabilities module / Inventory
- Filter an under evaluation document
- Verify the document details value is consistent with the filter applied

### Check List
